npm/css-src/sass/_multipleDropdown.scss

```diff
@@ -0,0 +1,207 @@
+.govuk-multi-autocomplete__wrapper {
```

Comment:
Has this SASS been copied from a source library?

Reply:
@thanhz OK, would you mind checking with Gareth if he copied this from somewhere. If so we should put a comment in the file with the URL to where this SASS comes from.

Reply:
I've put a comment in the SASS file with the link.